docs/release-notes/release-notes.md

```diff
@@ -62,6 +62,36 @@ ttl-num-blocks = 12
 max-gossip-delay = "1m0s"
 ```
 
+### Preparing for 128MB/6s
+
+This release introduces new CPU feature checks and updated hardware recommendations in preparation for future network upgrades to 128MB/6s blocks.
+**These requirements are not necessary for the current 32MB/6s block configuration
```

Comment:
[nit] we need to submit a gov proposal after v6 activates so the current block configuration on mainnet is still 8MB/6s. To avoid confusion, we can drop the "current"

```suggestion
**These requirements are not necessary for the 32MB/6s block configuration
```